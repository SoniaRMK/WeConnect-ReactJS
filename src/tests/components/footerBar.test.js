import {shallow} from 'enzyme';
import React from 'react';
import FooterBar from '../../components/navBar/footerBar';

describe('FooterBar component',() => {

    it('should match snapshot',() =>{
        const wrapper = shallow(<FooterBar/>)
        expect(wrapper).toMatchSnapshot();
    })
})
