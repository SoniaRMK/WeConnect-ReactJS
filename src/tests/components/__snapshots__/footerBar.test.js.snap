// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterBar component should match snapshot 1`] = `
<div
  className="footerBar"
>
  <hr />
  <div
    className="row"
  >
    <div
      className="col-xl-12"
    >
      <ul
        className="nav nav-pills nav-justified"
      >
        <li
          className="nav-item"
        >
          <a
            className="nav-link"
            style={
              Object {
                "color": "#17a2b8",
              }
            }
          >
            Â© 2018 WeConnect.
          </a>
        </li>
        <li
          className="nav-item"
        >
          <a
            className="nav-link"
            style={
              Object {
                "color": "#17a2b8",
              }
            }
          >
            Terms of Service
          </a>
        </li>
        <li
          className="nav-item"
        >
          <a
            className="nav-link"
            style={
              Object {
                "color": "#17a2b8",
              }
            }
          >
            Privacy Policy
          </a>
        </li>
      </ul>
      <br />
    </div>
  </div>
</div>
`;
